index.js

```diff
@@ -1 +1,220 @@
 console.log("Hello!!!");
+
+//Unique values
+// Check if word in anagrams arr is anagram
+// If you found anagram store this word in set or map
+// Values in set or map should not repeat anagram of this word
+// Also should not be present in set or map
+// Try to optimize script by using check has
+// Return collection set or map
+
+// Example: "actor" and "carot" are anagrams so store only "actor"
+
+const anagrams = [
+  "actor",
+  "carot",
+  "listen",
+  "enlist",
+  "debit",
+  "bidet",
+  "tear",
+  "rate",
+  "night",
+  "thing",
+  "lives",
+  "veils",
+  "stressed",
+  "desserts",
+  "dormitory",
+  "dirty room",
+  "rescue",
+  "secure",
+  "a gentleman",
+  "elegant man",
+  "listen",
+  "heart",
+  "angel",
+  "leader",
+  "silent",
+];
+
+const unique = (arr) => {
+  let x;
+  let y;
+  let z;
+  let litters = new Set();
+  let uniqueAnagrams = [];
+
+  for (let i = 0; i < arr.length; i++) {
+    x = arr[i].split('');
+    y = x.sort();
+    z = y.join("");
+
+    if (!litters.has(z)) {
+      litters.add(z);
+      uniqueAnagrams.push(arr[i]);
+    }
+
+  }
+  return uniqueAnagrams;
+
+};
+
+console.log(unique(anagrams));
+
+
+//Getter and setter
+// Create an user object
+// User should have to first name last name and age Add userInfo getter which gets all info from obj
+// Add userInfo setter to set all user info from string or obj
+// String setter format 'Taras Samoilenko 25'
+
+const user = {
+  firstName: "",
+  lastName: "",
+  age: "",
+  get userInfo(){
+    return `${this.firstName} ${this.lastName} is ${this.age}`
+  },
+  set userInfo(a){
+    if (typeof(a) === 'string') {
+      let arrWords = a.split(" ");
+      this.firstName = arrWords[0];
+      this.lastName = arrWords[1];
+      this.age = +arrWords[2];
+    } else if (typeof(a) === 'object') {
+      this.firstName = a.firstName;
+      this.lastName = a.lastName;
+      this.age = a.age;
+    }
+  },
+};
+
+user.userInfo = "Taras Samoilenko 25";
+user.userInfo;
+// "Taras Samoilenko is 25"
+
+user.userInfo = { firstName: "Kate", lastName: "Karp", age: 22 };
+user.userInfo;
+// "Kate Karp is 22"
+
+
+//Create arr method
+// Create obj with property from and to
+// Create function createArr that reads obj properties and
+// Creates new sorted arr with range from and to
+// Assign this arr as obj property
+
+// Create 3 solutions Solution 1 without bind, call or apply
+// Solution 2 with bind Solution 3 with call or apply
+
+const obj = {
+  from: 1,
+  to: 10,
+};
+
+const createArr = function (){
+  let arrFromTo = [];
+  for (let i = this.from; i <= this.to; i++){
+    arrFromTo.push(i);
+  };
+  return arrFromTo;
+};
+
+// Sol 1
+obj.createArr = createArr;
+
+// Sol 2
+const bindedSolution = createArr.bind(obj);
```

Comment:
that's correct but don't forget to call functions
`obj.createArr()`
`bindedSolution()`